test/house_test.rb

```diff
@@ -0,0 +1,130 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/room'
+require './lib/house'
+
+class HouseTest < Minitest::Test
+
+  def test_it_exists
+    house = House.new("$400000", "123 sugar lane")
+    assert_instance_of House, house
+  end
+
+  def test_has_readable_attributes
+    house = House.new("$400000", "123 sugar lane")
+    assert_equal 400000, house.price
+    assert_equal "123 sugar lane", house.address
+    assert_equal [], house.rooms
+  end
+
+  def test_it_can_add_rooms
+    house = House.new("$400000", "123 sugar lane")
+    room_1 = Room.new(:bedroom, 10, '13')
+    room_2 = Room.new(:bedroom, 11, '15')
+    house.add_room(room_1)
+    house.add_room(room_2)
+    expected = [room_1, room_2]
+    assert_equal expected, house.rooms
+  end
+
+  def test_above_market_average_false
+    house = House.new("$400000", "123 sugar lane")
+    refute house.above_market_average?
+  end
+
+  def test_above_market_average_true
+    house = House.new("$800000", "123 sugar lane")
+    assert house.above_market_average?
```

Comment:
similar to my note above - assert will pass if the method returns anything other than `false` or `nil`.  Definitely better to `assert_equal true` here.